fmriprep/cli/run.py

```diff
@@ -119,12 +119,16 @@ def get_parser():
         '--template', required=False, action='store',
         choices=['MNI152NLin2009cAsym'], default='MNI152NLin2009cAsym',
         help='volume template space (default: MNI152NLin2009cAsym)')
-
     g_conf.add_argument(
         '--output-grid-reference', required=False, action='store', default=None,
         help='Grid reference image for resampling BOLD files to volume template space. '
              'It determines the field of view and resolution of the output images, '
              'but is not used in normalization.')
+    g_conf.add_argument(
+        '--medial-surface-nan', required=False, action='store', default=False,
+        help='Replace medial wall values with NaNs on surface gifti files. Only '
+        'performed for gifti files mapped to an fsaverage')
```

Comment:
GIFTI is caps. And these are functional GIFTI files (see section 9.0 of the spec), not surface.